### PR TITLE
major.minor.x

### PR DESCRIPTION
**Pull request by ahmadnassri**
feat(versioning): use major.minor.x for groupping patch versions, decreasing the amount of work needed for releasing a patch

### CODE REVIEW

File: jekyll.yml [hunk 1]
```diff
@@ -2,9 +2,6 @@
 name: Kong - Open-Source API and Microservice Management Layer
 url: https://getkong.org
 share_url: http://getkong.org
-repo: https://github.com/Mashape/kong
-docrepo: https://github.com/Mashape/getkong.org
```
Comment (ahmadnassri) [inline jekyll.yml]: this wasn't used anywhere

File: jekyll.yml [hunk 2]
```diff
@@ -2,9 +2,6 @@
 name: Kong - Open-Source API and Microservice Management Layer
 url: https://getkong.org
 share_url: http://getkong.org
-repo: https://github.com/Mashape/kong
```
Comment (ahmadnassri) [inline jekyll.yml]: was only used once

File: jekyll.yml [hunk 3]
```diff
@@ -2,9 +2,6 @@
...
-docrepo: https://github.com/Mashape/getkong.org
-documentation: docs
```
Comment (ahmadnassri) [inline jekyll.yml]: seemed redundant for this to be a variable.

File: app/_jekyll_plugins/versions.rb [hunk 4]
```diff
@@ -11,7 +11,7 @@ def generate(site)
       # Add a `version` property to every versioned page
       site.pages.each do |page|
         parts = Pathname(page.path).each_filename.to_a
-        if parts[0] == site.config["documentation"]
+        if parts[0] == "docs"
```
Comment (thibaultCha) [inline app/_jekyll_plugins/versions.rb]: was nicer to have it as a variable

File: jekyll.yml (hunk 2 above)
Comment (thibaultCha) [inline jekyll.yml]: still nice to have as a variable

File: jekyll.yml (hunk 1 above)
Comment (thibaultCha) [inline jekyll.yml]: was used before, could be used again

File: jekyll.yml (hunk 3 above)
Comment (thibaultCha) [inline jekyll.yml]: might be renamed one day

**Comment by thibaultCha** [comment]
Ok for the .x renaming

Not ok for removing the variables (except the unused one eventually, but I feel like there should be a link on the website pointing to this repo, so it shouldn't be removed but rather used). Those were all are variables in case of quick renaming or moving of the project.

**Comment by thibaultCha** [comment]
By the way releasing a patch version will still require an update of the `luarocks_version` property anyways.

File: app/_jekyll_plugins/versions.rb (hunk 4 above)
Comment (ahmadnassri) [inline app/_jekyll_plugins/versions.rb]: when I actually removed the variable from the `yml` file it broke, not exactly sure why, so I brought it back there, but forgot about this spot.

File: jekyll.yml (hunk 1 above)
Comment (ahmadnassri) [inline jekyll.yml]: we can bring it back when needed

File: jekyll.yml (hunk 3 above)
Comment (ahmadnassri) [inline jekyll.yml]: and risk breaking all our SEO? unlikely ...

**Comment by ahmadnassri** [comment]
> By the way releasing a patch version will still require an update of the luarocks_version property anyways.

yep, this just eliminates the need to redo the whole folder...

**Comment by ahmadnassri** [comment]
> but I feel like there should be a link on the website pointing to this repo, so it shouldn't be removed but rather used

I had to double check to make sure we do link to the repo somewhere ... and we do (phew!) but seems like the variable itself was never used...

so yeah, i do agree, we should keep the variable, and update any existing reference to use it.

**Comment by ahmadnassri** [comment]
@thibaultCha cleaned up a bit, please take another look

**Pull request merged**